apps/builder/app/builder/features/style-panel/shared/style-collapsible-section.tsx

```diff
@@ -0,0 +1,36 @@
+import { SectionTitle, SectionTitleLabel } from "@webstudio-is/design-system";
+import type { ReactNode } from "react";
+import {
+  CollapsibleSectionBase,
+  useOpenState,
+} from "~/builder/shared/collapsible-section";
+import type { StyleSource } from "./style-info";
+
+export const StyleCollapsibleSection = (props: {
```

Comment:
Style as in StylePanelCollapsibleSection? I think we could just call it CollapsibleSection and rely on the import